Bottom: 100bb215e713c2ebf7ec9d377f22d6942cf24b01
Top:    093d235294d83864ab548e34921f2d1e0728c2df
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-07 01:27:03 +0100

Refresh of add-support-for-a-poured-objec-I

---

diff --git a/src/action.c b/src/action.c
index ea40c9f..1d2a8a5 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1459,7 +1459,7 @@ NotifyMode (void)
 	    /* first notify, lookup object */
 	  case STATE_FIRST:
 	    Crosshair.AttachedObject.Type =
-	      SearchScreen (Note.X, Note.Y, POLYGON_TYPE,
+	      SearchScreen (Note.X, Note.Y, POUR_TYPE,
 			    &Crosshair.AttachedObject.Ptr1,
 			    &Crosshair.AttachedObject.Ptr2,
 			    &Crosshair.AttachedObject.Ptr3);
@@ -1509,7 +1509,7 @@ NotifyMode (void)
 		  Flags = ((PolygonType *)Crosshair.AttachedObject.Ptr2)->Flags;
 		  PolyToPoursOnLayer (PCB->Data, Crosshair.AttachedObject.Ptr1,
 				      result, Flags);
-		  RemoveObject (POLYGON_TYPE,
+		  RemoveObject (POUR_TYPE,
 				Crosshair.AttachedObject.Ptr1,
 				Crosshair.AttachedObject.Ptr2,
 				Crosshair.AttachedObject.Ptr3);
